### ISSUE TITLE
Accessibility: No option to skip repeated content

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#266]
**Description**
A means of bypassing blocks of content that are repeated on multiple web pages must be provided.

**How to fix it:**
One of the ways to ensure this is to have a link at the top of each page that goes directly to the main content area. (The link should be the first focusable control on the web page and be visible when it receives focus). Such a link is not present on the page. If you have not ensured this by providing this type of links or by other means, this criterion is failed. Another way of ensuring conformance is to provide a heading for each block of content that are repeated on multiple web pages.

The scan shows all pages are affected by this error.

![3](https://user-images.githubusercontent.com/15264315/46361023-abf28280-c63a-11e8-89f9-7bac89401005.PNG)

**Comment by d4mation** [comment]
This has been pushed to Staging.

**Comment by banksashleym** [comment]
Looks great!

**Comment by d4mation** [comment]
This change has been pushed to Live 👍
